Fail to build rpm on centos-stream-10

Run 'make rpm' on centos-stream-10, but got this error:

+ export CC
+ CXX=g++
+ export CXX
+ cd greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f
+ /usr/bin/env CARGO_HOME=.cargo 'RUSTFLAGS=-Copt-level=3 -Cdebuginfo=2 -Ccodegen-units=1 -Cstrip=none -Clink-arg=-specs=/usr/lib/rpm/redhat/redhat-package-notes' /usr/bin/cargo build -j4 --profile rpm
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
warning: `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
error: could not load Cargo configuration

Caused by:
  could not parse TOML configuration in `/root/greenboot-rs/rpmbuild/BUILD/greenboot-7663c9ea12ba95bfccc5c14c89b9c50e5a17c14f/.cargo/config`

Caused by:
  TOML parse error at line 4, column 13
    |
  4 | rustflags = %{__global_rustflags_toml}
    |             ^
  invalid string
  expected `"`, `'`
error: Bad exit status from /var/tmp/rpm-tmp.Dsrd0B (%build)

RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.Dsrd0B (%build)
make: *** [Makefile:55: rpm] Error 1

--- a/greenboot.spec
+++ b/greenboot.spec
@@ -27,7 +27,7 @@ Source1:        %{name}-%{version}-vendor.tar.gz
 
 ExcludeArch:    s390x i686 %{power64}
 
-%if 0%{?rhel} && !0%{?eln}
+%if 0%{?centos} && !0%{?eln}
 BuildRequires:  rust-toolset
 %else
 BuildRequires:  rust-packaging
@@ -182,11 +182,7 @@ cat >.cargo/config << EOF
 [build]
 rustc = "%{__rustc}"
 rustdoc = "%{__rustdoc}"
-%if 0%{?rhel} && !0%{?eln}
-rustflags = %{__global_rustflags_toml}
-%else
 rustflags = "%{__global_rustflags_toml}"
-%endif
 
 [profile.rpm]
 inherits = "release"

